MIPX/MIPX.md

```diff
@@ -20,103 +20,111 @@ Replaces: N/A
 
-A subproposal is not required for an agreed modification of control.
-
-In the event an asset has not yet been 'claimed' by any controller, the new controller must still follow the requirements listed in this component.
+**Transparency**  
+An agreed modification of control must be made publicly accessible to Maker Governance using the MakerDAO discourse forum. It must remain publicly accessible for at least one week before being considered valid and actioned by the relevant Asset Owner.
 
-In order for a modification of control to be valid under this component, it must meet the following requirements:
+The agreed Modification of Management must use the template located at **[MIPXc3-ForumPost-Template.md](MIPXc3-ForumPost-Template.md)**.
 
-**Transparency**  
-An agreed mofication of control must be made publicly accessible to Maker Governance using the MakerDAO discourse forum. It must remain publicly accessible for at least one week before being considered valid and actioned by the relevant 'Owner-Controller'.
+**Well Defined**  
+An agreed Modification of Management must include a listed purpose for the asset and reasoning for why the change in management is desirable. Additionally, the new managers must provide a public contact address (email) and commit to responding promptly to contact requests from lawful authorities and mandated actors.
 
-The agreed modification of control must use the template located at **[MIPXc3-ForumPost-Template.md](MIPXc3-ForumPost-Template.md)**.
+**Recorded**
+An agreed Modification of Management must include a link to an open pull request on the official MIPs GitHub repository which modifies MIPXc2 to take into account the agreed modification. The link to this pull request must be present in the discourse forum post as part of transparency requirements. 
 
 **Supported**  
-An agreed modification of control must be publicly supported by at least three Core Unit Facilitators as defined in MIP41. 
+An agreed Modification of Management must be publicly supported by at least three Core Unit Facilitators as defined in MIP41. 
 
-In the case that a digital asset is currently controlled by one or more actors, the controlling actor(s) must support the modification of control in order for it to be valid under this component. 
+In the case that an Intangible Asset is currently managed by one or more actors, the managing actor(s) must support the Modification of Management in order for it to be valid under this component.
 
 **Uncontested**  
-An agreed modification of control must not be publicly contested by any Core Unit Facilitators as defined in MIP41.
-
-An agreed modification of control must not be publicly contested by Maker Governance. Maker Governance can use MIP24c4 to express contention quickly.
+An agreed Modification of Management must not be publicly contested by any of the following:
```

Comment:
Possible (wrong) alternative reading: "these following actors are excluded from being able to contest."

Reply:
Helpful, thanks.